doc/zmq.md

```diff
@@ -84,6 +84,7 @@ For instance:
 
     $ bitcoind -zmqpubhashtx=tcp://127.0.0.1:28332 \
                -zmqpubhashtx=tcp://192.168.1.2:28332 \
+               -zmqpubhashblock=tcp://[::1]:28333 \
```

Comment:
nit , quote arg `='...'`?

Reply:
Done! (with double instead of single quotes)